addons/stock/models/stock_move.py

```diff
@@ -458,11 +458,24 @@ def _compute_json_forecast(self):
                 else:
                     move.json_forecast = json.dumps({'expectedDate': None})
 
-    def _set_date_deadline(self):
+    def _set_date_deadline(self, new_deadline):
```

Comment:
we could use a kwargs?

Reply:
Yes why not.

Reply:
Well, I don't see the improvement to do that?

Reply:
Not needed. I though you call it with False specificaly